src/components/AssetsFilters/container.jsx

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
```

Comment:
This is just pulling the mapStateToProps, mapDispatchToProps, and connect out of AssetsFilters/index.jsx into AssetsFilters/container.jsx; there's no new code